docs/example-app/script/index.js

```diff
@@ -97,7 +98,7 @@ define([
         if (state === 'WAITING') {
           playbackElement.appendChild(playbackSpinner);
         } else {
-          if (playbackElement.contains(playbackSpinner)) playbackElement.removeChild(playbackSpinner);
+          DOMHelpers.safeRemoveElement(playbackSpinner);
         }
```

Comment:
the master branch has had the example app removed, so it might be worth merging that in to your branch.